articles/introduction-to-c#/index .md

```diff
@@ -0,0 +1,231 @@
+```
+
+### C# Syntax
+
+From the above code, we created a program that prints out "Hello World". Let us look at each line while modifying our program:
+
+```C#
+//This is a C# program
+using System;
+
+  namespace HelloWorld
+    {
+        class Program
+            {
+            static void Main(string[] args){
+                Console.WriteLine("Hello World!");
+                Console.ReadLine();
+                }
+            }
+        }
+```
+Here:
+
+
+**Line 1:** 
+
+```c#
+//This is a C# program
+```
+
+The line starts with two forward slashes which makes it a comment. The compiler ignores this line when compiling the code. This is a single-line comment. Comments are used by programmers to explain more about the line(what the line does). This concept is so important when a group of programmers are working together as it makes your code more readable to the others.
+
+For multiple-line comment,we use ` /*`and ends with `*/` :
+
+```C#
+/* C# is a programming language used to create Web applications and Mobile applications.
+
+It is an amazing language. */
+```
+
+Any text between `/*` and `*/` will be ignored by C#.
+
+**Line 2:**
+ ```C#
+ using System
+ ```
+
+In C#, it means that we can use the classes available in the system. It contains commonly-used types and classes. 
+You can explore more in the [C# docs](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive).
+
+**Line 3:**
+ ```C#
+ namespace HelloWorld{}
+ ``` 
+
+Namespaces are used in C# to seperate and organize code. They are containers consisting of methods, classes and other namespaces.
+
+**Line 4:**
+ ``` C#
+ class Program{}
+ ```
+
+A class is a container that contains methods and data. Since C# is Object-oriented programming, creating a class is mandatory for each program. In our case, we are creating a class called `Program`.
+
+**Line 5:** 
+```C#
+static void Main(string[] args)
+```
+Execution of programs begins from the `Main` method. Every C# program must have the `Main` method.
+
+**Line 6:**
+ ```C#
+ Console.WriteLine("Hello World!");
+ ```
+
+`Console.WriteLine()` is used to print out text on the console.It prints a string and moves to the next line. In our case, it is used to output *"Hello World"*.
+
+We can also use `Console.Write()` that print a string in the same line without moving to the next line.  You can learn more on Console.WriteLine() and Console.Write() [here](https://www.programiz.com/csharp-programming/basic-input-output).
+
+**Line 7** 
+```C#
+Console.ReadLine();
+```
+
+`Console.ReadLine()` is used to read the next characters in the input system.
+In the above program, as soon as the user enters a character(s) and presses the ENTER key, it terminates.
+
+`Console.Read()`can also be used to read the next character in the input system.
+
+```Console.ReadKey()``` can also be used to obtain the next key entered by the user. It is mostly used to hold the console for the user until he presses a key.
```

Comment:
Even though they are similar, they do different things. So instead of saying `can also be used`, rewrite it as `Console.Read() is used to ... ` and `Console.ReadKey() is used to ... `

Also, stress out the point that Console.Read() reads only one character from the input system.